spec/features/expense/expense_integration_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+
+def sign_in(user)
+  visit new_user_session_path
+  fill_in 'Email', with: user.email
+  fill_in 'Password', with: user.password
+  click_button 'Log in'
+end
+
+RSpec.describe 'Expense', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:user1) do
+    User.create(name: 'John', email: 'xyz@gmail.com', password: '123456', role: 'Admin')
+  end
+
+  before do
+    @user2 = User.create(name: 'Lily', email: 'abc@gmail.com', password: '123456')
+    @group1 = Group.create(name: 'Morgage', user: user1)
+    @group2 = Group.new(name: 'Transport', user: user1)
+    @group1.icon.attach(
+      io: File.open(Rails.root.join('app', 'assets', 'images', 'mo.png')),
```

Comment:
- [ ] Good job for making the required changes however, we still have some failing testcases, Kindly resolve these issues as tests ensure our code works as expected.

![image](https://github.com/wacoo/smart-budget-ror-capstone/assets/81932478/dbb766d4-0e08-4df8-ac67-b95129046098)

Context
It looks like you are encountering an error related to the version of ChromeDriver in your Selenium configuration for testing. The error indicates that the specified version, 119.0.6045, is not found on the ChromeDriver storage.